### PR TITLE
[Docs] CustomCommands Cog Guide

### PR DESCRIPTION
**Pull request by bobloy**
### Type

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature

### Description of the changes

Adds the customcom cog guide to the docs. Based on the descriptions provided in the commands.
Same method as #4488 , so depends on that approach being approved.

### CODE REVIEW

**Comment by bobloy** [comment]
Now ready for review. Part of #1734 as well.

I am interested in feedback on how I handled command aliases. I'm not sure Tip is the right way, or in the right spot.

File: docs/cog_guides/customcommands.rst [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+Examples:
+    - ``[p]customcom cooldown pingrole``
+    - ``[p]customcom cooldown yourcommand 30``
+    - ``[p]cc cooldown mycommand 30 guild``
+
+**Arguments:**
+
+- ``<command>`` The custom command to check or set the cooldown.
+- ``<cooldown>`` The number of seconds to wait before allowing the command to be invoked again. If ommited, will instead return the current cooldown settings.
```
Comment (Flame442) [inline docs/cog_guides/customcommands.rst]:
```suggestion
- ``<cooldown>`` The number of seconds to wait before allowing the command to be invoked again. If omitted, will instead return the current cooldown settings.
```

File: redbot/cogs/customcom/customcom.py [hunk 2]
```diff
@@ -387,8 +413,16 @@ async def cc_cooldown(
...
-        - `[p]customcom cooldown yourcommand 30`
+        Examples:
+            - `[p]customcom cooldown pingrole`
+            - `[p]customcom cooldown yourcommand 30`
+            - `[p]cc cooldown mycommand 30 guild`
+
+        **Arguments:**
+
+        - `<command>` The custom command to check or set the cooldown.
+        - `<cooldown>` The number of seconds to wait before allowing the command to be invoked again. If ommited, will instead return the current cooldown settings.
```
Comment (Flame442) [inline redbot/cogs/customcom/customcom.py]:
```suggestion
        - `<cooldown>` The number of seconds to wait before allowing the command to be invoked again. If omitted, will instead return the current cooldown settings.
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
One (1) typo, then this should be good to go!

**Review by Flame442 — APPROVED** [review]
Thanks!

**Pull request merged**